policy/openbot/models.py

```diff
@@ -72,7 +72,7 @@ def pilot_net(img_width, img_height, bn=False):
                      bn=bn)
     
     # fuse input MLP and CNN
-    combinedInput = tf.keras.layers.concatenate([mlp.input, cnn.output])
+    combinedInput = tf.keras.layers.concatenate([mlp.output, cnn.output])
```

Comment:
Actually here, we should take the input directly. No need for the MLP. This is since pilot_net does not have a MLP for conditional command input. But in practice both should work. If we keep mlp.output we should probably use a proper MLP also.

Reply:
I assumed it is a mistake, every other `combinedInput` uses `mlp.output`

Reply:
You are right, it does kinda look like a bug. It was just to reproduce pilot_net for the paper with minimal changes. But I guess for the repo we could adapt it.